src/parse/ast.rs

```diff
@@ -20,7 +20,8 @@ pub enum ManipulationStmt {
 	Update(UpdateStmt),
 	Select(SelectStmt),
 	Insert(InsertStmt),
-	Delete(DeleteStmt)
+	Delete(DeleteStmt),
+    Use(UseStmt)
 }
```

Comment:
the types of your enum are to far indented.
Check in your Sublime editor that the tab Width is set to 4 spaces, 
you can see that in the bottom right in sublime.